airbyte-integrations/connectors/source-alloydb-strict-encrypt/src/main/java/io/airbyte/integrations/source/alloydb/AlloyDbStrictEncryptSource.java

```diff
@@ -0,0 +1,18 @@
+package io.airbyte.integrations.source.alloydb;
+
+import static io.airbyte.integrations.source.relationaldb.state.StateManager.LOGGER;
+
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.base.Source;
+import io.airbyte.integrations.source.postgres.PostgresSourceStrictEncrypt;
+
+public class AlloyDbStrictEncryptSource {
+
+  public static void main(String[] args) throws Exception {
```

Comment:
should we somehow modify spec for this connector?

Reply:
No, because we wrap `PostgresSourceStrictEncrypt` which already modifies a spec. On the UI it has the following look:
![screen](https://user-images.githubusercontent.com/35812734/190666046-3d8f1c28-87fc-4af2-9071-1b6c6f471740.png)